devtools/projects/ng-devtools/src/lib/devtools.component.scss

```
@@ -215,3 +215,23 @@
     color: #000;
   }
 }
+
+.ng-dev-mode-causes {
+  font-weight: 500;
+  font-size: 1.2em;
+  padding: 1rem;
+  width: 80%;
+  margin: auto;
+  border: 1px solid;
+  border-radius: 16px;
+
+  code {
+    padding: 1px;
+    color: lightgreen;
+    background: #3e3e3e;
+  }
```

```suggestion
  code {
    padding: 2px;
    color: lightgreen;
    background: #3e3e3e;
    border-radius: 5px;
  }
```

I guess it will feel more natural to have the code part as a pill.